simulation/groups_interface.py

```diff
@@ -4,117 +4,47 @@
 
 
 from abc import abstractmethod, ABCMeta
+from itertools import count
+import random
 
 
 class PopulationGroup(metaclass=ABCMeta):
     """
     Abstract class for specific population groups based on SIR model.
     """
+
+    id = count(0)
```

Comment:
Can you explain why base class also defines id? Every child class will override this.

Reply:
I implement this auto labeling convention prefix_id but I am not sure if we need this in further visualisation or simple prefix will be enough? I thought that id should be defined in base class or constructor do not work correctly.